frontend/workflows/projectSelector/src/project-selector.tsx

```diff
@@ -171,9 +177,42 @@ const ProjectSelector = () => {
       <StateContext.Provider value={derivedState}>
         <StyledSelectorContainer>
           <StyledWorkflowHeader>
-            {/* TODO: change icon to match design */}
-            <LayersIcon />
+            <LayersOutlinedIcon />
             <StyledWorkflowTitle>Dash</StyledWorkflowTitle>
+            <Tooltip
+              title={
+                <>
+                  <TooltipContainer>
+                    <Typography variant="subtitle3" color="#FFFFFF">
+                      Projects
+                    </Typography>
+                    <Typography variant="body3" color="#E7E7EA">
+                      {projectDescription}
+                    </Typography>
+                  </TooltipContainer>
+                  <TooltipContainer>
+                    <Typography variant="subtitle3" color="#FFFFFF">
+                      Upstreams
+                    </Typography>
+                    <Typography variant="body3" color="#E7E7EA">
+                      {upstreamDescription}
+                    </Typography>
+                  </TooltipContainer>
+                  <TooltipContainer>
+                    <Typography variant="subtitle3" color="#FFFFFF">
+                      Downstreams
+                    </Typography>
+                    <Typography variant="body3" color="#E7E7EA">
+                      {downstreamDescription}
+                    </Typography>
+                  </TooltipContainer>
+                </>
+              }
+              placement="right-start"
+              maxWidth="400px"
+            >
+              <InfoOutlinedIcon />
```

Comment:
icon is too large

Reply:
from the designs, the 2 icons seem to be the same size so I made them both smaller
![Screen Shot 2021-07-30 at 12 56 55 PM](https://user-images.githubusercontent.com/39421794/127686660-0e7e17ec-ca6d-49f3-ba07-a477d14d12d9.png)

The design specifies 18px but it looks too small IMO
![Screen Shot 2021-07-30 at 12 50 46 PM](https://user-images.githubusercontent.com/39421794/127686759-02af24c6-80cc-4af6-99a4-8cb14ac6c09b.png)

I brought them down to `fontSize=small` instead which is 20px. LMK what you think
![Screen Shot 2021-07-30 at 12 50 54 PM](https://user-images.githubusercontent.com/39421794/127686850-ed52b900-5fb3-4979-888c-1414c64a8a29.png)